<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyWindow</class>
 <widget class="QDialog" name="StudyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#StudyWindow{background-color: rgb(255, 213, 214)}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QListWidget" name="listArtist">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <widget class="QPushButton" name="next">
        <property name="styleSheet">
         <string notr="true">QPushButton#next {
  color: rgb(23, 23, 23); /* цвет текста */
  background: rgb(112, 112, 112); /* фон кнопки */
  outline: none; 
} 
#next:hover { background: rgb(133, 133, 133); } /* при наведении курсора мышки */
</string>
        </property>
        <property name="text">
         <string>Следующая</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="previous">
        <property name="styleSheet">
         <string notr="true">QPushButton#previous {
  color: rgb(23, 23, 23); /* цвет текста */
  background: rgb(112, 112, 112); /* фон кнопки */
  outline: none; 
} 
#previous:hover { background: rgb(133, 133, 133); } /* при наведении курсора мышки */
</string>
        </property>
        <property name="text">
         <string>Предыдущая</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Segoe Print</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Название картины</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="yearLabel">
        <property name="font">
         <font>
          <family>Segoe Print</family>
         </font>
        </property>
        <property name="text">
         <string>Года</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="windowTitleLabel">
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Название</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
